### PR TITLE
💚 Use the same conditions on `send_tweet` as `release`

### PR DESCRIPTION
**Pull request by Kludex**
Selected Reviewer: @adriangb

### CODE REVIEW

**Review by samuelcolvin — APPROVED** [review]
LGTM, i think we could have needed `if : "always() && needs.release.status == 'success'`, but this is safer.

**Review by samuelcolvin — CHANGES_REQUESTED** [review]
actually this won't work as it needs the output from `release`.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -420,9 +420,10 @@ jobs:
 
   send-tweet:
     name: Send tweet
+    needs: [check, publish_docs]
+    # `always()` is required to trigger this task even though test-fastapi is skipped
+    if: "always() && needs.check.outputs.result == 'success' && startsWith(github.ref, 'refs/tags/')"
```
Comment (samuelcolvin) [inline .github/workflows/ci.yml:425]:
```suggestion
    needs: [release]
    # `always()` is required to trigger this task even though test-fastapi is skipped
    if: "always() && needs.release.status == 'success'"
```

Comment (Kludex) [inline .github/workflows/ci.yml:425]:
```suggestion
    needs: [release]
    # `always()` is required to trigger this task even though test-fastapi is skipped
    if : "always() && needs.release.status == 'success'"
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -420,9 +420,10 @@ jobs:
 
   send-tweet:
     name: Send tweet
-    runs-on: ubuntu-latest
     needs: [release]
-    if: ${{ needs.release.status == 'success' }}
+    # `always()` is required to trigger this task even though test-fastapi is skipped
+    if : "always() && needs.release.status == 'success'"
```
Comment (Kludex) [inline .github/workflows/ci.yml:425]:
```suggestion
    if: "always() && needs.release.status == 'success'"
```

**Pull request merged**